definitions/divorce/json/CaseTypeTab/CaseTypeTab-general-referral-nonprod.json

```diff
@@ -6,9 +6,19 @@
     "TabID": "generalReferral",
     "TabLabel": "General Referral",
     "TabDisplayOrder": 24,
-    "CaseFieldID": "GeneralReferralReason",
+    "CaseFieldID": "GeneralReferrals",
     "TabFieldDisplayOrder": 1
   },
+  {
```

Comment:
Do we really need the fields below? It seems to me that these are the fields used to generate an individual general referral. All this data should be in the collection, shouldn't it?

Reply:
Oh, ok. Got you. There's more than one event between creating a general referral and adding it to the collection, is that right?

Reply:
Yup